Formula/surfraw.rb

```diff
@@ -20,6 +13,13 @@ class Surfraw < Formula
     sha256 "30b885dc5908318868da2739f36834ce071bc7bff1a761fdc395afe82f75efa8" => :mountain_lion
   end
 
+  head do
+    url "https://anonscm.debian.org/git/surfraw/surfraw.git"
```

Comment:
Good question: No, I didn't. I'd rather put it back, if that may be an important parameter.